os_migrate/roles/conversion_host/tasks/conv_hosts_inventory.yml

```diff
@@ -4,9 +4,7 @@
     inventory_name: "{{ item.inventory_name }}"
     inventory_group: "{{ item.inventory_group }}"
     os_migrate_conversion_host_name: "{{ item.os_migrate_conversion_host_name }}"
-    os_migrate_conversion_auth: "{{ item.os_migrate_conversion_auth }}"
-    os_migrate_conversion_auth_type: "{{ item.os_migrate_conversion_auth_type }}"
-    os_migrate_conversion_region_name: "{{ item.os_migrate_conversion_region_name }}"
+    cloud: "{{ os_migrate_conversion_cloud }}"
```

Comment:
Here we need to pass the variable through to the inner tasks. The original pattern is:

```
os_migrate_conversion_auth: "{{ item.os_migrate_conversion_auth }}"
```

so we should be doing:


```
os_migrate_conversion_cloud: "{{ item.os_migrate_conversion_cloud }}"
```

In addition, grepping reveals that there are a few spots missed in the replacement:

```
@ ~/openstack/repos/os-migrate
$ grep -rin os_migrate_conversion_auth os_migrate/
os_migrate/roles/conversion_host/tasks/delete.yml:7:    auth: "{{ os_migrate_conversion_auth }}"
os_migrate/roles/conversion_host/tasks/delete.yml:8:    auth_type: "{{ os_migrate_conversion_auth_type|default(omit) }}"
os_migrate/roles/conversion_host/tasks/delete.yml:19:    auth: "{{ os_migrate_conversion_auth }}"
os_migrate/roles/conversion_host/tasks/delete.yml:20:    auth_type: "{{ os_migrate_conversion_auth_type|default(omit) }}"
os_migrate/roles/conversion_host/tasks/delete.yml:31:    auth: "{{ os_migrate_conversion_auth }}"
os_migrate/roles/conversion_host/tasks/delete.yml:32:    auth_type: "{{ os_migrate_conversion_auth_type|default(omit) }}"
```

Otherwise this looks good, i think we're getting close :)